mod/aalborg_theme/views/default/css/elements/components.php

```diff
@@ -204,10 +204,13 @@
 .elgg-river-attachments .elgg-icon-arrow-right {
 	margin: 3px 8px 0;
 }
-.elgg-river-layout .elgg-input-dropdown {
+.elgg-river-layout .elgg-river-selector {
```

Comment:
I don't think we can actually remove .egg-input-dropdown for BC reasons. If someone replace river/filter view, this will not take effect.